coreneuron/network/multisend_setup.cpp

```diff
@@ -118,39 +118,37 @@ static int* newintval(int val, int size) {
     return x;
 }
 
-static int* newoffset(int* acnt, int size) {
-    int* aoff = new int[size + 1];
+static std::vector<int> newoffset(const std::vector<int>& acnt) {
```

Comment:
As you are going through the trouble of changing all these functions and their signatures, can you please start adding doxygen with a `\brief` and a description of the arguments and return?

Reply:
Of course ;)

Reply:
In fact there is already a lot of comments everywhere